subsys/bluetooth/audio/Kconfig.has

```diff
@@ -33,6 +39,18 @@ config BT_HAS_PRESET_NAME_DYNAMIC
 	help
 	  Enabling this option allows for runtime configuration of preset name.
 
+config BT_HAS_PRESET_CONTROL_POINT_NOTIFIABLE
+	bool "Preset Control Point Notifiable support"
+	depends on BT_HAS_PRESET_SUPPORT && BT_EATT
+	help
+	  This option enables support for clients to subscribe for notifications
+	  on the Hearing Aid Preset Control Point characteristic.
+
+config BT_HAS_ACTIVE_PRESET_INDEX
+	def_bool BT_HAS_PRESET_SUPPORT
+	help
+	  This option enables the Hearing Aid Active Preset Index characteristic.
```

Comment:
Technically you are right, but I just think its neater to show this under its own Kconfig than shoving it in under presets, since it is conditional on Preset Control Point.